src/__init__.py

```diff
@@ -1,18 +1,43 @@
-from flask import Flask, request, jsonify
+from flask import Flask, request, jsonify, g
 from src.models.tagging_suggester import *
 import nltk
+from datetime import datetime
+import src.utils.app_config as app_config
+from models import Request
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
 
 app = Flask(__name__)
 
 tagging_suggester = TaggingSuggester()
 
+@app.before_request
+def before_request():
+    engine = create_engine(app_config.database_url(), strategy='threadlocal')
+    Session = sessionmaker(bind=engine)
+    g.db_session = Session()
+
 @app.route('/create', methods=['POST'])
 def tagging_suggestion():
     json = request.get_json()
-    print(json)
-    result = tagging_suggester.predict(json['text'])
+    request_record = Request(
+        created_at = datetime.now().isoformat(),
```

Comment:
Is this value consistent with what the database expects (a string in the format `YYYY-MM-DD`)?

Reply:
Also not a requirement to be specified as you have opted for a default in the creation of the table, if you didn't want to specify it.

Reply:
As discussed